src/fides/api/service/saas_request/saas_request_override_factory.py

```diff
@@ -213,5 +226,17 @@ def validate_consent_override_function(f: Callable) -> None:
         )
 
 
+def validate_get_consentable_item_function(f: Callable) -> None:
+    pass
+
+
+def validate_update_consent_function(f: Callable) -> None:
+    pass
+
+
+def validate_process_consent_webhook_function(f: Callable) -> None:
+    pass
```

Comment:
Not adding the validation functions just yet, this might change as we go through the implementation. To avoid having to change the implementations and tests, we'll save this for the end.